src/nimble.nim

```diff
@@ -1129,10 +1129,8 @@ proc run(options: Options) =
   # Build the binary.
   build(options)
 
-  # Now run it.
-  let args = options.action.runFlags.join(" ")
-
-  doCmd("$# $#" % [binaryPath, args], showOutput = true)
+  let args = options.action.runFlags.map(x => x.escape).join(" ")
```

Comment:
```suggestion
  let args = options.action.runFlags.map(x => x.quoteShell).join(" ")
```

`os.quoteShell` is much better for quoting things to pass to the shell.

Reply:
ah, thanks. I hadn't seen that proc.

Looking at the docs, I also now see `quoteShellCommand`. I'm assuming it would be appropriate to use that instead of the current `quoteShell`, `join`, and `%`?